src/components/combo/combo.ts

```diff
@@ -325,6 +326,19 @@ export default class IgcComboComponent<T extends object>
     return html`${this.groupKey && item[this.groupKey]}`;
   };
 
+  /**
+   * Sets the component's positioning strategy.
+   */
+  @property({ attribute: 'position-strategy' })
+  public positionStrategy: 'absolute' | 'fixed' = 'fixed';
```

Comment:
```suggestion
  public positionStrategy: 'absolute' | 'fixed' = 'absolute';
```
Can we keep the old behavior?

Reply:
As far as I understood from @damyanpetev it doesn't work with absolute in the context of the grid!?